### ISSUE TITLE
[BUG]: Time series compare_models() returning empty list

### ISSUE THREAD

**Issue by samaelizabeth** [pycaret/pycaret#3530]
### pycaret version checks

- [X] I have checked that this issue has not already been reported [here](https://github.com/pycaret/pycaret/issues).

- [X] I have confirmed this bug exists on the [latest version](https://github.com/pycaret/pycaret/releases) of pycaret.

- [X] I have confirmed this bug exists on the master branch of pycaret (pip install -U git+https://github.com/pycaret/pycaret.git@master).

### Issue Description

While running the tutorial code with 'airlines' dataset, compare_models() returns empty score grid.
![image](https://user-images.githubusercontent.com/15118872/236317222-35ba731f-45e1-4857-83b9-e18d58a89b16.png)

### Reproducible Example

```python
from pycaret.datasets import get_data
airline = get_data('airline')
from pycaret.time_series import *
exp_name = setup(data = airline,  fh = 12, verbose=False)
best_model = compare_models()
best_model
```

### Expected Behavior

Even create_model('naive') is not working for me. Expected behaviour is a score grid with all values

### Actual Results

```python-traceback
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-15-bd158155b764> in <cell line: 6>()
      4 exp_name = setup(data = airline,  fh = 12, verbose=False)
      5 best_model = compare_models()
----> 6 predict_model(best_model)

1 frames
/usr/local/lib/python3.10/dist-packages/pycaret/time_series/forecasting/oop.py in predict_model(self, estimator, fh, X, return_pred_int, alpha, coverage, round, verbose)
   4434         """
   4435 
-> 4436         estimator.check_is_fitted()
   4437 
   4438         pipeline_with_model, estimator_ = self._predict_model_reconcile_pipe_estimator(

AttributeError: 'list' object has no attribute 'check_is_fitted'
```

### Installed Versions

<details>
3.0.0
</details>

**Comment by ngupta23** [comment]
Please add the output of the show_versions command so we can try to reproduce this.

```python
from pycaret import show_versions
show_versions()
```

**Comment by ngupta23** [comment]
If you are using a version of sktime > 0.17.0, can you try with 0.17.0 instead. I think that might fix your issue.

**Comment by federicodecillia** [comment]
Hi, I had the same issue, and with sktime=0.17.0 it fixed. Thanks

**Comment by samaelizabeth** [comment]
Thanks, issue got resolved with with sktime=0.17.0

**Comment by robertasgabrys** [comment]
Thank you, sktime=0.17.0 solves the problem.
